README.md

```diff
@@ -19,3 +19,211 @@ Requires:
+
+### Step 4.1: GitHub Personal Access Token (PAT)
+
+Create a Personal Access Token (PAT) for your GitHub account via the page https://github.com/settings/tokens where you find a button "Generate new token".
+Give it meaningful name (field titled "Note") and set the expiration date. Then select the scopes this PAT will be used for. Then click "Generate token". On the result page, take note/copy the resulting token string -- it will only be shown once.
+
+On the Linux machine set the environment variable `GITHUB_TOKEN`, e.g.
+```
+export GITHUB_TOKEN='THE_TOKEN_STRING'
+```
+
+### Step 4.2: GitHub App Secret Token
+The GitHub App Secret Token is used to verify the webhook sender. You should have created one already when registering a new GitHub App in Step 1.
+
+On the Linux machine set the environment variable `GITHUB_APP_SECTRET_TOKEN`, e.g.
+```
+export GITHUB_APP_SECRET_TOKEN='THE_SECRET_TOKEN_STRING'
+```
+Note, depending on the characters used in the string you will likely have to use single quotes when setting the value of the environment variable.
+
+### Step 4.3: Create a private key and store it on the Linux machine
+The private key is needed to let the app authenticate when updating information at the repository such as commenting on PRs, adding labels, etc. You can create the key at the page of the GitHub App you have registered in Step 1.
+
+Open the page https://github.com/settings/apps and then click on the icon left to the name of the GitHub App for the EESSI bot or the "Edit" button for the app. Near the end of the page you will find a section "Private keys" where you can create a private key by clicking on the button "Generate a private key". The private key should be automatically downloaded to your local computer. Copy it to the Linux machine and set the environment variable as follows
+```
+export GITHUB_PRIVATE_KEY="$(cat PATH_TO_PRIVATE_KEY_FILE)"
+```
+
+### Step 4.4: Create a directory containing all necessary scripts
+
+#### After [PR#10](https://github.com/EESSI/eessi-bot-software-layer/pull/10) has been merged
+
+The bot just needs one script `eessi-bot-build.slurm` available from the `scripts` directory in the repository [EESSI/eessi-bot-software-layer](https://github.com/EESSI/eessi-bot-software-layer) (or your fork of it) and an `app.cfg` that points to the location of that script on your system.
+
+First, clone the EESSI/eessi-bot-software-layer repository (or your fork of it) by running
+
+```
+git clone https://github.com/EESSI/eessi-bot-software-layer.git
+```
+
+Change directory with `cd eessi-bot-software-layer` and note the full path of the directory (`pwd`).
+
+Add a configuration file (if it doesn't exist yet) by running
+
+```
+cp -i app.cfg.example app.cfg
+```
+
+Inside `app.cfg` change the value of `build_job_script` to `FULL_PATH/scripts/eessi-bot-build.slurm`
+
+#### Before [PR#10](https://github.com/EESSI/eessi-bot-software-layer/pull/10) has been merged
```

Comment:
This whole block should be removed, just assume that this PR won't be merged before #10 is merged (and mention that #10 should be merged first in the description of this PR right here)

Reply:
Rewrote this part. Removed what is obsolete since the merge of #10